cobra/cmd/project.go

```diff
@@ -75,6 +75,7 @@ func (p *Project) createLicenseFile() error {
 	if err != nil {
 		return err
 	}
+	defer licenseFile.Close()
```

Comment:
Good catch!

Reply:
Thanks! It was actually hard to spot because all the CI tests pass, but this was making them fail in a `golang` docker container on a Windows 10 host. I'm yet not sure about this 'fix' being required for all versions of golang; or whether it is specific to windows hosts. Nevertheless, I believe it is better to have it.